fcsa_service_request-1.0.0-resolved.json

```diff
@@ -0,0 +1,571 @@
+        }
+      },
+      "Session_Schema": {
+        "required": [
+          "session_id"
+        ],
+        "properties": {
+          "session_id": {
+            "type": "string",
+            "example": "k;X/`[AP\\K9Bc<H^vgCv_F.J8w9/%~+V"
+          }
+        }
+      },
+      "Request_Attributes_Schema": {
+        "required": [
+          "account_number",
+          "address",
+          "category_1",
+          "category_2",
+          "comm",
+          "email",
+          "firstName",
+          "id",
+          "lastName",
+          "latitude",
+          "longitude",
+          "message",
+          "street",
+          "street_number",
+          "subtype",
+          "suburb",
+          "telephone",
+          "type",
+          "username"
+        ],
+        "type": "object",
+        "properties": {
+          "id": {
+            "type": "integer",
+            "format": "int64",
+            "example": 1
+          },
+          "category_1": {
+            "type": "string",
+            "example": "Support COMPLAINTS"
+          },
+          "category_2": {
+            "type": "string",
+            "example": "Street Lights - Single Light Out"
+          },
+          "suburb": {
+            "type": "string",
+            "example": "ADRIAANSE"
+          },
+          "email": {
+            "type": "string",
+            "example": "cdorsett3@loc.gov"
+          },
+          "message": {
+            "type": "string",
+            "example": "This requests was captured by the Walking the Streets program."
+          },
+          "address": {
+            "type": "string",
+            "example": "08 Maple Wood Hill"
+          },
+          "username": {
+            "type": "string",
+            "example": ""
+          },
+          "account_number": {
+            "type": "string",
+            "example": ""
+          },
+          "comm": {
+            "type": "string",
+            "example": "Maple"
+          },
+          "latitude": {
+            "type": "string",
+            "example": "-34.1080381288507"
+          },
+          "longitude": {
+            "type": "string",
+            "example": "18.47152590751648"
+          },
+          "type": {
+            "type": "integer",
+            "example": 1005
+          },
+          "subtype": {
+            "type": "integer",
+            "example": 1001
+          },
+          "telephone": {
+            "type": "string",
+            "example": ""
+          },
+          "firstName": {
+            "type": "string",
+            "example": "Gwenora"
+          },
+          "lastName": {
+            "type": "string",
+            "example": "Mellor"
+          },
+          "street": {
+            "type": "string",
+            "example": "Utah"
+          },
+          "street_number": {
+            "type": "integer",
+            "example": 76
+          }
+        }
+      },
+      "Request_Attributes_Detail_Schema": {
```

Comment:
Hmm, this is a little confusingly named, relative to `Request_Attributes_Schema`. Maybe rather `Request_Attributes_Lookup_Schema`?